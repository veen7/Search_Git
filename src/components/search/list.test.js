import React, { Component, addons } from 'react';
import { connect } from 'react-redux';
import { shallow } from 'enzyme';
import List from './list';


function setup() {
	const props = {
			search : []
	}
	const enzymeWrapper = shallow(<List {...props} />)

  return {
    props,
    enzymeWrapper
  }
}


describe('components', () => {
  	describe('render', () => {
	    it('render component', () => {
	    	 const { enzymeWrapper } = setup();
	    	 expect(enzymeWrapper).toMatchSnapshot();
	
  		})
	})
})

