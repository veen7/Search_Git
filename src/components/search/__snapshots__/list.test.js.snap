exports[`components render render component 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <ul>
    
</ul>,
  "nodes": Array [
    <ul>
      
</ul>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance":  {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <List
        search={Array []} />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {
          "search": Array [],
        },
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <ul>
          
</ul>,
        "_debugID": 2,
        "_renderedOutput": <ul>
          
</ul>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <List
    search={Array []} />,
}
`;
